project/Dependencies.scala

```diff
@@ -150,7 +152,7 @@ object Dependencies {
   val commonsExec = "org.apache.commons" % "commons-exec" % "1.3" force()
   val commonsCodec = "commons-codec" % "commons-codec" % "1.10" force()
 
-  val defaultGuavaVersion = sys.props.getOrElse("guava.version", "14.0.1") // 16.0.1 for cassandra connector 1.6-M1
+  val defaultGuavaVersion = sys.props.getOrElse("guava.version", "16.0.1") // 16.0.1 for cassandra connector 1.6-M1
```

Comment:
Spark is pretty decoupled from that now, the hard deps on 14 is on old versions (AFAIK).
So it can be anything, 16 is good for cassandra which is the main blocker that I've seen so far

sounds good?